sw/host/tests/chip/pwrmgr/src/sleep_all_wakeups.rs

```
@@ -28,33 +28,42 @@ fn sleep_all_wakeups_test(opts: &Opts, transport: &TransportWrapper) -> Result<(
     loop {
         let vec = UartConsole::wait_for(
             &*uart,
-            r"PASS|FAIL|Issue WFI to enter sleep ([0-9]+)",
+            r"PASS|Test ([0-9]+) begin \(([^)]*)\)",
```

Is this correct? In my mind this regex should be `r"PASS|FAIL|Test ([0-9]+) begin \(([^)]*)\)"`, but perhaps I'm missing something here. I see that you match `FAIL` on the next UART console synchronisation, but the `pwrmgr_sleep_all_wake_ups` test could still fail between entry to `test_main` and the first `execute_test` call, I think?

---

Yes that's a good point, it doesn't harm to catch a `FAIL` if that ever happened. I added back FAIL.